recipes/qt/6.x.x/conanfile.py

```diff
@@ -775,7 +775,8 @@ def _create_plugin(pluginname, libname, type, requires):
             _create_module("WaylandClient", ["Gui", "wayland::wayland-client"])
             _create_module("WaylandCompositor", ["Gui", "wayland::wayland-server"])
 
-        self.cpp_info.components["qtCore"].cxxflags.append("-fPIC")
+        if self.settings.compiler != "Visual Studio":
+            self.cpp_info.components["qtCore"].cxxflags.append("-fPIC")
```

Comment:
```suggestion
        if self.settings.os != "Windows":
            self.cpp_info.components["qtCore"].cxxflags.append("-fPIC")
```

Reply:
by the way, why this flag in package_info()?

Reply:
this avoids falling into https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/global/qglobal.h#n1394